R/zzz.R

```diff
@@ -44,6 +44,44 @@ default_linters <- modify_defaults(
 #' There is a list for the default elements and another that contains all available elements.
 #' Use [modify_defaults()] to produce a custom list.
 #'
+#' @details
+#' The following functions can be regarded as undesirable:
```

Comment:
are considered undesirable by default? not sure we currently convey that here

Reply:
The catch is we have `default_undesirable_functions`, which is a proper subset of `all_undesirable_functions`.
Documentation currently contains remarks on all, not just on the defaults.

Reply:
Got it... maybe "The following functions are often regarded as undesirable:" ?

Reply:
We even use `attach()` occasionally, to attach some helper functions for interactive use in a `.Rprofile`.

Reply:
`return()` is one that definitely doesn't make sense to me -- I'd be fine with dropping it as undesirable. probably the better approach is to delegate that to the upcoming `explicit_return_linter`, which will use tidyverse style by default (throw a lint on a terminal `return()` statement)

for attach, i agree there's definitely valid use cases, but they are rare (?) compared to the "beginner" mistake of using it to try and manage the search path

Reply:
Yep, I was suggesting to keep these. I'd still weaken the statement due to the fact that `library()` is ubiquitous in non-package projects.